docs/rules/prefer-enabled-disabled.md

```diff
@@ -12,12 +12,12 @@ Consider these 2 snippets:
 
 ```js
 const { getByRole } = render(<input type="checkbox" disabled />);
-const element = getByRole('checkbox');
-expect(element).toHaveProperty('disabled'); // passes
+const element = getByRole("checkbox");
+expect(element).toHaveProperty("disabled"); // passes
 
 const { getByRole } = render(<input type="checkbox" />);
-const element = getByRole('checkbox');
-expect(element).toHaveProperty('disabled'); // also passes 😱
+const element = getByRole("checkbox");
+expect(element).toHaveProperty("disabled"); // also passes 😱
```

Comment:
Huh, why did these get double-quotted if this project is using the kcd-scripts prettier config?

Reply:
Doh, I forgot to add the dodd in front of the husky & prettier config file 😅